packages/graphql/mock-server/mixin.core.js

```diff
@@ -210,9 +214,17 @@ module.exports = class GraphQLMockServerMixin extends Mixin {
       const server = new ApolloServer({
         schema: this.schema,
         context: ({ req }) => ({ req, config: this.config }),
+        playground: {
+          settings: {
+            'request.credentials': 'include',
```

Comment:
For what exactly is this needed? Which request will then include credentials?

Reply:
This is for the Request from the playgound (former graphiQL) to theserver.